### ISSUE TITLE
Topic list post snippets handle quotes wrong

### ISSUE THREAD

**Issue by loopback0** [apxltd/what-bugs#20]
The post snippet from the topic list:
![image](https://cloud.githubusercontent.com/assets/8740474/13936291/7e5acdf0-efb4-11e5-8c41-eaf7d24dd82e.png)

The post:
![image](https://cloud.githubusercontent.com/assets/8740474/13936293/81b895cc-efb4-11e5-9855-6a58231d60aa.png)

"blakeyrat said" shouldn't show.

**Comment by sjennison** [comment]
I love the example :laughing:

**Comment by sjennison** [comment]
Also, if this is too hard to handle, perhaps replace the quote with `quote` or something?

So it would end up appearing

> [avatar] less than a minute ago
> @[user] said: `quote`
> [response]

**Comment by sjennison** [comment]
Using my mad paint skillz...

![image](https://cloud.githubusercontent.com/assets/5776850/13942515/88d892f2-efc9-11e5-9707-90c68dc1a75b.png)

**Comment by coldandtired** [comment]
D'oh! I posted it on the forum a few days ago but this is how long it takes me to get anything useful done :)

**Comment by sjennison** [comment]
FWIW, this is partially mitigated by the post-context hack - it inserts the link and makes it clear that it's a quote. Not ideal, because you lose the quote context, but it's better than it was

**Comment by loopback0** [comment]
Nah. While the "blah said in topic whatever" is an improvement,  it now takes up more of the teaser.
If the "blah says in topic" thing was inside the quote, like most other forum software, then it'd not be an issue.

**Comment by LB--** [comment]
I have been manually adding a > before the `@blah said:` to move it inside the quote, but I don't know if this silences the notification or not.

**Comment by BenLubar** [comment]
It does silence the notification.
